packages/cli/generation/ir-generator/src/__test__/test-definitions/enum.json

```diff
@@ -2570,7 +2570,7 @@
                     "autogeneratedExamples": [
                         {
                             "example": {
-                                "url": "/path/",
+                                "url": "/path/>/>/red/red",
```

Comment:
just noticed this and looking into it

Reply:
spec here:

```yml
service:
  auth: false
  base-path: ""
  endpoints:
    send:
      path: /path/{operand}/{maybeOperand}/{operandOrColor}/{maybeOperandOrColor}
      method: POST
      path-parameters: 
        operand: root.Operand
        maybeOperand: optional<root.Operand>
        operandOrColor: root.ColorOrOperand
        maybeOperandOrColor: optional<root.ColorOrOperand>
      examples: 
        - path-parameters:
            operand: $root.Operand.GreaterThan
            maybeOperand: $root.Operand.LessThan
            operandOrColor: $root.ColorOrOperand.Red
            maybeOperandOrColor: $root.ColorOrOperand.Red

types: 
  Operand:
    docs: | 
      Tests enum name and value can be 
      different.
    enum:
      - value: ">"
        name: GREATER_THAN 
      - value: "="
        name: EQUAL_TO
      - value: "less_than"
        docs: | 
          The name and value should be similar
          are similar for less than. 
    examples:
      - name: GreaterThan
        value: ">"
      - name: LessThan
        value: "less_than"
  
  Color:
    enum:
      - value: "red"
        name: RED 
      - value: "blue"
        name: BLUE
    examples:
      - name: Red
        value: "red"
  
  ColorOrOperand:
    discriminated: false
    union:
      - Color
      - Operand
    examples:
      - name: Red
        value: "red"
```

Reply:
added url encoding to path paramters. otherwise wire tests would fail for this test